### PR TITLE
Don't expose broken or fragile APIs to plugins

### PR DESCRIPTION
**Pull request by sergiusens**
LP: #1571264

Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

**Comment by sergiusens** [comment]
I've exploratory tested this to death; I hope either @kyrofa or @elopio approve :-)

File: snapcraft/common.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015, 2016 Canonical Ltd
+# Copyright (C) 2015 Canonical Ltd
```
Comment (kyrofa) [inline snapcraft/common.py]: This should probably still be 2016, no?

Comment (sergiusens) [inline snapcraft/common.py]: @kyrofa true, it should be only 2016 though as it is completely new

**Comment by kyrofa** [comment]
Yeah this looks great :+1:

**Pull request merged**